cura/Settings/GlobalStack.py

```diff
@@ -83,6 +83,15 @@ def supportsMaterialExport(self):
         """
         return self.getMetaDataEntry("supports_material_export", False)
 
+    @pyqtProperty("QVariantList", constant = True)
```

Comment:
Isn't there a specific tag for methods ? This is not really a property

Reply:
I don't think you can set `constant` on a method. This optimizes it a bit, and I also think that since it is constant, that from the perspective of QML, you _can_ actually see it as a property.